assets/javascripts/discourse/admin/models/ai-persona.js

```diff
@@ -23,6 +23,13 @@ const ATTRIBUTES = [
   "rag_uploads",
 ];
 
+// rag params are populated on save, only show it when editing
```

Comment:
Yeah the annoying thing is that you actually want to do this before the first file is uploaded. I wanted to avoid having to carry the defaults on the client side, but I guess we can.

Reply:
Chunking only happens after saving/updating so you can display the values on file upload, or if the persona has files. Maybe we can include them in the serializer's meta hash?